### PR TITLE
BLADE-491 kotlin plugin included in gradle 5

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
ci:retest

**Comment by lawrence-lee** [comment]
ci:retest

**Comment by lawrence-lee** [comment]
ci:test:72

**Comment by lawrence-lee** [comment]
@gamerson, can you review these changes? These test failures need to be fixed in project templates, but i think we can go ahead with this.

File: gradle/apps/service-builder/jndi/jndi-service/build.gradle [hunk 1]
```diff
@@ -31,31 +25,4 @@ buildService {
 	apiDir = "../jndi-api/src/main/java"
 }
 
-group = "com.liferay.blade.samples.jndiservicebuilder"
-
-startTestableTomcat {
-	checkTimeout = 10 * 60 * 1000
-}
-
-testIntegration {
```
Comment (gamerson) [inline gradle/apps/service-builder/jndi/jndi-service/build.gradle:40]: Does this remove integration testing for this module?

**Review by gamerson — APPROVED** [review]
LGTM 👍

File: gradle/apps/service-builder/jndi/jndi-service/build.gradle (hunk 1 above)
Comment (lawrence-lee) [reply gradle/apps/service-builder/jndi/jndi-service/build.gradle:40]: Yes, but testing still exists in the liferay-workspace version of the module

**Comment by lawrence-lee** [comment]
sent here: https://github.com/lawrence-lee/liferay-blade-samples/pull/73

**Pull request closed without merging**